internal/storage/memory.go

```diff
@@ -156,18 +255,21 @@ func (s *memoryIssuerService) createTables() error {
 func (s *memoryIssuerService) insertIssuer(iss types.Issuer) error {
 	q := `
         INSERT INTO issuers (
-            id, name, uri, jwksuri, mappings
+            %s
         ) VALUES
-        ($1, $2, $3, $4, $5);
+        ($1, $2, $3, $4, $5, $6);
```

Comment:
Adding this means that querying for a URI can return more than one issuer row.  If there were the same issuer with two different tenant IDs, then it's possible the `Create.GetByURI` test case fails.

Reply:
I added a global uniqueness constraint for URI to deal with that for now.